examples/flutter_gallery/lib/welcome/steps/documentation_step.dart

```diff
@@ -0,0 +1,111 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_gallery/welcome/step.dart';
+
+const String _kGalleryAssetsPackage = 'flutter_gallery_assets';
+
+class DocumentationWelcomeStep extends WelcomeStep {
+  DocumentationWelcomeStep({TickerProvider tickerProvider})
+      : super(tickerProvider: tickerProvider);
+
+  AnimationController _animationController;
+  AnimationController _quickAnimationController;
+  Animation<double> _barScaleAnimation;
+  Animation<double> _barOpacityAnimation;
+  Animation<double> _focusScaleAnimation;
+  Animation<double> _focusOpacityAnimation;
+  Animation<double> _iconOpacityAnimation;
+
+  @override
+  String title() => 'Complete, flexible APIs';
+  @override
+  String subtitle() =>
+      'View full API documentation, when you need it, with a quick tap. Look for the documentation icon in the app bar.';
+
+  @override
+  Widget imageWidget() {
+    _animationController = AnimationController(
+      vsync: tickerProvider,
+      duration: Duration(milliseconds: 300),
+    );
+    _quickAnimationController = AnimationController(
+      vsync: tickerProvider,
+      duration: Duration(milliseconds: 120),
+    );
+    _barScaleAnimation =
+        Tween<double>(begin: 1.0, end: 0.95).animate(_animationController);
+    _barOpacityAnimation =
+        Tween<double>(begin: 1.0, end: 0.8).animate(_quickAnimationController);
+    _focusScaleAnimation = Tween<double>(begin: 0.4, end: 1.0).animate(
+      CurvedAnimation(parent: _animationController, curve: Curves.easeInOut),
+    );
+    _focusOpacityAnimation = Tween<double>(begin: 0.0, end: 1.0).animate(
+      CurvedAnimation(parent: _animationController, curve: Curves.easeInOut),
+    );
+    _iconOpacityAnimation =
+        Tween<double>(begin: 1.0, end: 0.0).animate(_quickAnimationController);
+
+    final Image barImage = Image.asset(
+      'welcome/welcome_documentation.png',
```

Comment:
Is the resolution right on this? Looks a bit blurry on iPhone.

Reply:
I'll try to get another one pushed up